packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/integ.job-submission-workflow.ts

```diff
@@ -0,0 +1,79 @@
+import * as eks from '@aws-cdk/aws-eks';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import {
+  Classification, VirtualClusterInput, EksClusterInput, EmrContainersDeleteVirtualCluster,
+  EmrContainersEksCreateVirtualCluster, EmrContainersStartJobRun, ReleaseLabel,
+} from '../../lib';
+
+/**
+ * Stack verification steps:
+ * Everything in the link below must be setup before running the state machine.
+ * @see https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up.html
+ *
+ * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * aws stepfunctions describe-execution --execution-arn <exection-arn generated before> : should return status as SUCCEEDED
+ */
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-stepfunctions-tasks-emr-containers-all-services-integ');
+
+const eksCluster = eks.Cluster.fromClusterAttributes(stack, 'EKS Cluster', {
+  clusterName: 'test-eks',
+});
+
+const createVirtualCluster = new EmrContainersEksCreateVirtualCluster(stack, 'Create a virtual Cluster', {
+  virtualClusterName: 'Virtual-Cluster-Name',
+  eksNamespace: 'spark2',
+  eksCluster: EksClusterInput.fromCluster(eksCluster),
+  resultPath: '$.cluster',
+});
+
+const startJobRunJob = new EmrContainersStartJobRun(stack, 'Start a Job Run', {
+  virtualCluster: VirtualClusterInput.fromTaskInput(sfn.TaskInput.fromJsonPathAt('$.cluster.Id')),
+  releaseLabel: ReleaseLabel.EMR_6_2_0,
+  jobName: 'EMR-Containers-Job',
+  executionRole: iam.Role.fromRoleArn(stack, 'Job-Execution-Role', 'arn:aws:iam::070850498885:role/Job-Execution-Role'),
```

Comment:
should also create a role, not use an existing one

Reply:
That's means we'd have to go through the whole trust role policy update.

Reply:
Yes. Detail these instructions at the top of this file